### DISCUSSION TITLE
Thanks Kuo Kuo for sharing this PicoBETH project!

### DISCUSSION THREAD

**Discussion by tommy9112** [206cc/PicoBETH#4] [General]
This is my production
![微信图片_20240824015722](https://github.com/user-attachments/assets/e702f376-5ec1-48bb-bad7-e75a366ae899)
![微信图片_20240824015729](https://github.com/user-attachments/assets/d76f329f-7478-4579-a0ea-7425d30e976e)
v1
![微信图片_20240824015735](https://github.com/user-attachments/assets/4a0cf8af-d36b-4344-954f-5ffc46265451)
![微信图片_20240824015742](https://github.com/user-attachments/assets/8cf90a24-65ea-4da0-b52d-e8027fabcd59)
v2

Thank you very much

**Comment by 206cc** [comment]
Wow, you've done an amazing job! I can tell that a lot of thought went into the turntable. The V1 version looks very sturdy, but it seems difficult to make, and my tools can't handle making it. You've done a better job than I could. Additionally, I suggest moving the bead clamping head closer to the sliding table, as this might help extend the platform's lifespan.

May I ask which country you're from?

**Comment by tommy9112** [comment]
![微信图片_20240902102701](https://github.com/user-attachments/assets/bb8dfe23-3c8a-4132-a69a-89edb470167e)
![微信图片_20240902102707](https://github.com/user-attachments/assets/f429076e-f434-400f-8c52-a871dbde72c5)
v3

**Comment by SSNQ** [comment]
太酷了，我很想像你们一样拥有那么强的动手能力！我会穿线，但是没有那么多的时间可以让我复刻你们的案例！谢谢 kuokuo 的分享！真的太厉害了

**Reply by 206cc** [reply]
蠻好玩的，成本也不高

**Comment by tommy9112** [comment]
最近在搞这个触摸开关，如果能实现还是比较方便。但是目前是经常误触不知道是灵敏太高还是什么情况[
![微信图片_20241020235351](https://github.com/user-attachments/assets/3f66a679-25d8-4ad2-a4ae-63a8767ccebb)
![微信图片_20241020235345](https://github.com/user-attachments/assets/1ea5c508-d491-47cf-a68a-19402cc5765d)
![微信图片_20241020235330](https://github.com/user-attachments/assets/6efdb608-c439-4aa5-bc62-0bd1c89edfc5)
![微信图片_20241020235323](https://github.com/user-attachments/assets/71931714-b8bd-44e8-916f-c35fe4b8a70a)
](url)

**Reply by tommy9112** [reply]
也不算误触，就是经常自己就启动

**Reply by 206cc** [reply]
看起來不錯，這個觸控也便宜，已下訂回來試試。

**Reply by tommy9112** [reply]
网上说223的用交流电源会有误触发的情况，用电池会改善。
我已下定另外一个，也是ttp223 但是上面的电容电阻等零件多点可能抗干扰会强点，然后也买了100pf的电容来调低灵敏度。

**Reply by 206cc** [reply]
你也可以試試這個，我用起來沒毛病很順手
[imp/beadclip-btn@jpliew](https://github.com/206cc/PicoBETH/tree/imp/beadclip-btn%40jpliew)

**Comment by SSNQ** [comment]
能请大佬代为制作吗？咱们可以商量工时费和材料费吗？

**Reply by 206cc** [reply]
目前在弄網球版本 & V2架構，還沒什麼時間。

看有沒有朋友會做機械加工的，能幫一下忙，應該是加工需要一些工具，其它就還好。

**Comment by tommy9112** [comment]
大佬，下个版本有空的话帮忙利用空余的引脚写几个按键  可以# 着，有需要的可以用下
1:  打结键
2：预设磅数1
3：预设磅数2（可以在代码中填好预设，当然如果能在菜单中设置那更专业了）
4：速度切换（循环递增）
我下个尝试可能更换薄膜按键
![微信图片_20241022020705](https://github.com/user-attachments/assets/a60694f8-76eb-4166-b1e3-0c630eb77b75)
![微信图片_20241022020700](https://github.com/user-attachments/assets/a6ab518b-4cb0-4f5b-902e-298fa9f49d0a)
![微信图片_20241022020654](https://github.com/user-attachments/assets/d456ec82-7082-494b-b382-095521265f4b)

**Reply by 206cc** [reply]
以上都做的到，但要等所有開發測試都結束了才能來做，你也可以試看看，不會很難。

按鍵開關的話我比較偏號機械按壓的回饋感覺XD

**Comment by hungitc1992** [comment]
I have an electronic stringer and it works great for me. But one thing I want to fix is ​​that the LBS on my machine is only 1lbs, I want to fix it to 0.5 lbs or as detailed as possible. Can you fix it? Please give me the fee. It would be great if you could edit the code remotely and get it done. The machine I am using is Leung S10 Plus
我有一个电子球拍张紧器，它对我来说非常有用。但我想要修复的一件事是我的机器上的 LBS 只有 1lbs，我想将其修复为 0.5 lbs 或尽可能详细。你能修复它吗？请给我费用。如果能够远程编辑代码并完成它那就太好了。我使用的设备是Leung S10 Plus
![image](https://github.com/user-attachments/assets/e37cfaf6-79b5-4282-88b8-c6912267ae7f)

**Reply by 206cc** [reply]
I checked, and this machine should support adjustments in 0.5 lbs increments. I recommend contacting the manufacturer’s after-sales service to confirm if there’s a malfunction.

Commercially available machines like this usually don’t allow for independent core program upgrades, and manufacturers typically don’t release the source code. If you need an upgrade or repair, it can only be done through official channels.

我查了一下，這台機器應該是可以 0.5 磅為單位進行調整的，建議您聯繫廠商的售後服務，確認是否存在故障。

市面上銷售的機器通常無法自行升級核心程序，廠商也不會公開源代碼。如果需要升級或維修，只能通過官方渠道進行。

**Reply by hungitc1992** [reply]
I bought the machine used. I asked to Leung but Leung never support because i dont have order code.
Thank Kuokuo, Have a nice day.
P/s: Stick with the PicoBETH. As a long time badminton stringer, I see so many great points in the Pico. It is much better than many high end machines on the market. Looking forward to the commercial version.
我买了二手机器。我问过故障，但故障从未得到支持，因为我没有订单代码。
谢谢 Kuokuo，祝你有美好的一天。
附言：坚持使用 PicoBETH。作为一名长期的羽毛球穿线师，我看到 Pico 有很多优点。它比市场上许多高端机器好得多。期待商业版。

**Reply by 206cc** [reply]
I don’t have the capability to commercialize it myself, but the license permits commercial use, so perhaps a manufacturer will commercialize it in the future. The purpose of this project is to help reduce the price of electronic stringing machines, as they are currently too expensive for many stringers.
我自己沒有商業化的能力，但許可證是允許商業用途的，或許未來會有廠商將其商業化。這個專案的目的是希望降低電子穿線機的價格，因為對許多穿線師來說，電子穿線機的價格實在過於昂貴。

**Comment by tommy9112** [comment]
终于更新了

**Reply by 206cc** [reply]
最近的更新都是安全檢查和修一些無關緊要的BUG，程式方面也沒什麼好改的了，接下來會慢慢補上以下說明，只是沒什麼時間可以寫..
1. 將一些HW2組件使用3D列印的降低製作難度(已測完)
2. 全3D列印版本(已測完)
3. 網球版本(已測完)
